axelar-chains-config/info/testnet.json

```diff
@@ -1714,6 +1714,61 @@
       "gasOptions": {
         "gasLimit": 8000000
       }
+    },
+    "immutable": {
+      "name": "Immutable",
+      "id": "immutable",
+      "chainId": 13473,
+      "rpc": "https://rpc-geth.testnet.immutable.com",
+      "tokenSymbol": "IMX",
+      "contracts": {},
+      "gasOptions": {
+        "gasLimit": 8000000,
+        "maxPriorityFeePerGas": 150000000000,
```

Comment:
This should include the same create2/3 deployer configs as well as they're shared with stagenet